app/assets/javascripts/comments.js

```diff
@@ -179,8 +179,10 @@ var Comments = (function() {
         var $el = $(this);
         var menu = $el.find('.dropdown-menu');
         var leftPos = $el.offset().left;
+        var parentTopPos = $el.offset().top;
         if (leftPos + menu.width() > $(window).width()) {
-          menu.offset({ left: leftPos - menu.width() });
+          menu.offset({ left: leftPos - menu.width(),
+                        top: (parentTopPos + 20)});
```

Comment:
Here you can use constants which has Matej created. You can find them or add new in config/initializers/constants.rb